### ISSUE TITLE
CocoaPods Spec

### ISSUE THREAD

**Issue by Ashton-W** [mattgallagher/CwlSignal#2]
I’ve created a `.podspec` for **CwlSignal** so it can be added to the [CocoaPods](https://github.com/CocoaPods/CocoaPods) package manager specs.

https://github.com/Ashton-W/CwlSignal

However, **CwlSignal** doesn't have any version tags. A version tag will make dependency resolution much easier.

[Semantic version](http://semver.org) tags (instead of plain commit hashes/revisions) allow for [resolution of cross-dependencies](https://github.com/CocoaPods/Specs/wiki/Cross-dependencies-resolution-example).

In case you didn’t know this yet; you can tag the current HEAD as, for instance, version 1.0.0, like so:

```
$ git tag -a 1.0.0 -m "Tag release 1.0.0"
$ git push --tags
```

Matt, please excuse my use of an issue template

**Comment by mattgallagher** [comment]
Hi Ashton,

Thanks for pointing this out. Adding a 1.0.0 tag was something I intended to do but it had slipped my mind. I've immediately added one to the repository.

You're welcome to shoot me a pull request for the podspec or otherwise let me know if there's anything else that would help you smooth out CocoaPods usage.

**Comment by mattgallagher** [comment]
Merged podspecs: f74eee90982769974f0038757fa81780b91a35b6
